infra/aws/cloudformation/iam/cdn.packages.k8s.io/cloudformation.yaml

```diff
@@ -0,0 +1,210 @@
+  ProvisionerUser:
+    Type: 'AWS::IAM::User'
+    Properties:
+      Path: /
+      UserName: provisioner
+      Policies:
+        - PolicyName: ProvisionerUserAccess
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Action: 'sts:AssumeRole'
+                Effect: Allow
+                Resource: !Sub 'arn:aws:iam::${AWS::AccountId}:role/Provisioner'
+
+  # ProvisionerRole is an IAM role assumed by provisioner IAM user to run Terraform
+  ProvisionerRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      RoleName: Provisioner
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Action: 'sts:AssumeRole'
+            Effect: Allow
+            Principal:
+              AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:user/provisioner'
+    DependsOn: ProvisionerUser
+
+  # ProvisionerPlanAccessPolicy is an IAM policy that grants permissions to run `terraform plan`
+  ProvisionerPlanAccessPolicy:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      ManagedPolicyName: ProvisionerPlanAccess
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: 'Allow'
+            Action:
+              - 'acm:DescribeCertificate'
+              - 'acm:ListTagsForCertificate'
+            Resource: !Sub 'arn:aws:acm:us-east-1:${AWS::AccountId}:certificate/*'
+          - Effect: 'Allow'
+            Action:
+              - 'cloudfront:GetCachePolicy'
+              - 'cloudfront:GetDistribution'
+              - 'cloudfront:GetOriginAccessControl'
+              - 'cloudfront:ListCachePolicies'
+              - 'cloudfront:ListTagsForResource'
+            Resource:
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:distribution/*'
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:cache-policy/*'
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:origin-access-control/*'
+          - Effect: 'Allow'
+            Action:
+              - 's3:GetAccelerateConfiguration'
+              - 's3:GetBucketAcl'
+              - 's3:GetBucketCORS'
+              - 's3:GetBucketLogging'
+              - 's3:GetBucketObjectLockConfiguration'
+              - 's3:GetBucketOwnershipControls'
+              - 's3:GetBucketPolicy'
+              - 's3:GetBucketRequestPayment'
+              - 's3:GetBucketTagging'
+              - 's3:GetBucketVersioning'
+              - 's3:GetBucketWebsite'
+              - 's3:GetEncryptionConfiguration'
+              - 's3:GetLifecycleConfiguration'
+              - 's3:GetReplicationConfiguration'
+              - 's3:ListBucket'
+            Resource: 'arn:aws:s3:::*-cdn-packages-k8s-io-*'
+          - Effect: 'Allow'
+            Action:
+              - 's3:GetObject'
+              - 's3:ListBucket'
+            Resource:
+              - !Sub 'arn:aws:s3:::${TFStateBucket}'
+              - !Sub 'arn:aws:s3:::${TFStateBucket}/*'
```

Comment:
I'd put the state access into a separate policy, not just a separate statement in the same policy.